iOS/Settings/General/SettingsRows.swift

```diff
@@ -49,15 +49,24 @@ struct SettingsRow {
 	/// This row, when tapped, will push the the Add Account screen
 	/// in to view.
 	static var addAccount: some View {
-		NavigationLink(destination: AccountsManagementView()) {
-			Label {
-				Text("button.title.manage-accounts", comment: "Manage Accounts")
-			} icon: {
-				Image("app.account")
-					.resizable()
-					.frame(width: 25.0, height: 25.0)
-					.clipShape(RoundedRectangle(cornerRadius: 6))
-			}
+//		NavigationLink(destination: AccountsManagementView()) {
+//			Label {
+//				Text("button.title.manage-accounts", comment: "Manage Accounts")
+//			} icon: {
+//				Image("app.account")
+//					.resizable()
+//					.frame(width: 25.0, height: 25.0)
+//					.clipShape(RoundedRectangle(cornerRadius: 6))
+//			}
+//		}
```

Comment:
Should we delete this code instead of commenting it out?

Reply:
I've fixed this in 17c4252.